pkg/controller/operators/olm/operator_test.go

```diff
@@ -1236,8 +1238,8 @@ func TestTransitionCSV(t *testing.T) {
 						"0.0.0",
 						"",
 						installStrategy("csv1-dep1", nil, nil),
-						[]*v1beta1.CustomResourceDefinition{crd("c1", "v1", "g1")},
 						[]*v1beta1.CustomResourceDefinition{},
```

Comment:
doesn't this change from owned to required? that seems odd for this test

Reply:
Meant to remove this change.